src/components/UserProfile/UserSettings/UserGeneralSettings/index.tsx

```diff
@@ -409,6 +419,99 @@ const UserGeneralSettings = () => {
                     </div>
                   </>
                 )}
+              {hasPermission(
+                [Permission.AUTO_REQUEST, Permission.AUTO_REQUEST_MOVIE],
+                { type: 'or' }
+              ) &&
+                user?.userType === UserType.PLEX && (
+                  <div className="form-row">
+                    <label
+                      htmlFor="watchlistSyncMovies"
+                      className="checkbox-label"
+                    >
+                      <span>
+                        {intl.formatMessage(messages.plexwatchlistsyncmovies)}
+                      </span>
+                      <span className="label-tip">
+                        {intl.formatMessage(
+                          messages.plexwatchlistsyncmoviestip,
+                          {
+                            PlexWatchlistSupportLink: (
+                              msg: React.ReactNode
+                            ) => (
+                              <a
+                                href="https://support.plex.tv/articles/universal-watchlist/"
+                                className="text-white transition duration-300 hover:underline"
+                                target="_blank"
+                                rel="noreferrer"
+                              >
+                                {msg}
+                              </a>
+                            ),
```

Comment:
Would it be better to define this once instead of duplicating it for both movies & TV? Like we do for the `OpenPgpLink` for email https://github.com/sct/overseerr/blob/5247f14968c1f5a1a2efdc26f850cf81c0c04f0e/src/components/Settings/Notifications/NotificationsEmail.tsx#L49-L55

Reply:
I don't think we need to. It's only used in two places and they are fairly close to each other.